### DISCUSSION TITLE
[Feature] Make mobile app more responsive

### DISCUSSION THREAD

**Discussion by temal32** [immich-app/immich#14565] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello,
It would be nice if Immich was more responsive on mobile phones. Especially for iPhones, it would be nice if you were able to switch between the tabs "Photos", "Search", "Albums" and "Library" only by wiping on your screen from Right to Left or Left to Right. And I don't know why, but in general the iOS version of Immich feels very "heavy", it doesn't feel fast like any other app developed for iOS. Additionally it would be super cool if you were able to remove the "Photos, Search, Albums, Library" Text from the bottom of the app, so only the icons stay (provided as in the screenshot). This would make the app look "cleaner" and I'd preffer it, maybe this could be in the settings to remove the text.
![screee](https://github.com/user-attachments/assets/004aa1e8-f6cf-4d8f-92f5-651b07688ed7)

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is several feature requests in one. Please keep it to one per thread, and search for duplicates before creating them.

**Reply by temal32** [reply]
bo0tzz really being my biggest hater at this point

**Reply by alextran1502** [reply]
@temal32 I know you like to joke, but statement like that should not be the topic for the joke 😀

**Reply by temal32** [reply]
👍
